src/SFA.DAS.EmploymentCheck.AcceptanceTests/Steps/Hooks.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System.Threading.Tasks;
+using TechTalk.SpecFlow;
+using BoDi;
+using SFA.DAS.EmploymentCheck.SubmissionEventWorkerRole;
+using SFA.DAS.EmploymentCheck.AcceptanceTests.Infrastructure;
+using SFA.DAS.AccountsApiSubstitute.WebAPI;
+using SFA.DAS.EmploymentCheck.AcceptanceTests.DependencyResolution;
+using SFA.DAS.ProviderEventsApiSubstitute.WebAPI;
+using SFA.DAS.CommitmentsApiSubstitute.WebAPI;
+using SFA.DAS.HmrcApiSubstitute.WebAPI;
+using SFA.DAS.TokenServiceApiSubstitute.WebAPI;
+using SFA.DAS.EventsApiSubstitute.WebAPI;
+using SFA.DAS.NLog.Logger;
+
+namespace SFA.DAS.EmploymentCheck.AcceptanceTests.Steps
+{
+    [Binding]
+    public class Hooks
+    {
+        private ILog _log;
+
+        private IObjectContainer _objectContainer;
+
+        private WorkerRole sut;
+
+        private AccountsApi AccountsApi;
+
+        private ProviderEventsApi ProviderEventsApi;
+        
+        private CommitmentsApi CommitmentsApi;
+
+        private HmrcApi HmrcApi;
+
+        private EventsApi EventsApiSubstitute;
+        
+        private TokenServiceApi TokenApiSubstitute;
```

Comment:
I think I'd wrap the APIs in a containing class so the creation can be managed in a single place to reduce what this class is doing.  It's an optional change tho.